docker/actinia-stac-tests/actinia-stac-test.cfg

```diff
@@ -0,0 +1,30 @@
+[GRASS]
+grass_database = /actinia_core/grassdb
+grass_user_database = /actinia_core/userdata
+grass_tmp_database = /actinia_core/workspace/temp_db
+grass_resource_dir = /actinia_core/resources
+grass_gis_base = /usr/local/grass7
+grass_gis_start_script = /usr/local/bin/grass
+grass_addon_path = /root/.grass7/addons/
+
+[API]
+plugins = ["actinia_stac_plugin"]
+force_https_urls = True
+
+[REDIS]
+redis_server_url = localhost
+redis_server_port = 6379
+redis_queue_server_url = localhost
+redis_queue_server_port = 6379
+worker_queue_name = actinia_job
+worker_logfile = /actinia_core/workspace/tmp/actinia_worker_test.log
```

Comment:
These can be ommited. (`redis_queue_server_*` configs are deprecated, the others are not necessarily needed)

Reply:
When I delete the port declaration, the GitHubAction failed, the error mention the port is not found and can not be run. That's why I put it again.